e2e/cypress/integration/enterprise/profile_popover/profile_popover_spec.js

```diff
@@ -456,7 +456,7 @@ const verifyAddToChannel = (user, visible = true) => {
 const clickAddToChannel = (user) => {
     // # Open profile popover
     cy.get('#postListContent', {timeout: TIMEOUTS.ONE_MIN}).within(() => {
-        cy.findAllByText(user.username).first().should('have.text', user.username).click();
+        cy.findAllByText(`@${user.username}`).first().should('have.text', '@' + user.username).click();
```

Comment:
Multiple tests are failing for me on this step.

Reply:
Thanks for bringing this up Prapti! All tests passed on me (see screenshot) but upon looking closer, MM-T2927_4 script does not match with step 4 of MM-T2927 (Zephyr). Also, here in code there were 4 steps only while 5 in Zephyr. @furqanmlk please take a look again, thanks!
<img width="744" alt="Screen Shot 2021-03-12 at 5 21 08 PM" src="https://user-images.githubusercontent.com/5334504/110920578-849a5680-8358-11eb-9452-73315ce74731.png">

Reply:
@saturninoabril thanks for commenting.
Sure I am looking at this, I did not go in the details as it is in prod status. I was just fixing the test cases only.
Dark theme test case is missing.
Should I add the new test case in this PR or should I open new one ? this PR has fixes for other test cases.

Reply:
See https://github.com/mattermost/mattermost-webapp/pull/7676/files#r593212532.